app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/address/enter_uk_address.scala.html

```diff
@@ -42,16 +43,20 @@
   returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link
 )
 
-@(form: Form[UkAddress], backLink: Call, submit: Call, enterPostcode: Call, addressJourneyType: AddressJourneyType, isATrust: Boolean, representativeType: Option[Either[PersonalRepresentative.type, Capacitor.type]])(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
+@(form: Form[UkAddress], backLink: Call, submit: Call, enterPostcode: Call, addressJourneyType: AddressJourneyType, isATrust: Boolean, representativeType: Option[RepresentativeType])(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
 
 @key = @{"address"}
 @isAgent = @{ request.userType.contains(UserType.Agent) }
 @userKey = @{
-  if (representativeType.exists(_.isLeft)) ".personalRep"
-  else if (representativeType.exists(_.isRight)) ".capacitor"
-  else if (isAgent) ".agent"
-  else if (isATrust) ".trust"
-  else ""
+  representativeType match {
+      case Some(PersonalRepresentative) => ".personalRep"
+      case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRep"
```

Comment:
Are we going to use same "PersonalRepresentative" messages for "PersonalRepresentativeInPeriodOfAdmin"?

Reply:
no, but for now that's what it's going to be until we do the tickets to implement it properly for period of admin.